src/server/scripts/poller.js

```diff
@@ -82,7 +89,7 @@ export const pollRepositories = (checker) => {
 
           // Do not even check for changes if the snap was already built in the
           // last `POLLER_BUILD_THRESHOLD` interval (typically 24h).
-          const threshold = conf.get('POLLER_BUILD_THRESHOLD');
+          const threshold = parseInt(conf.get('POLLER_BUILD_THRESHOLD'));
```

Comment:
It would be safer to use `parseInt(conf.get('POLLER_BUILD_THRESHOLD'), 10)` to make sure JS is not trying to guess the base of the number based on contents.

Reply:
sure, fix on its way.